pontoon/base/models.py

```diff
@@ -290,12 +290,26 @@ def serialized_notifications(self):
         is_comment = False
 
         if hasattr(notification.actor, "slug"):
-            actor = {
-                "anchor": notification.actor.name,
-                "url": reverse(
-                    "pontoon.projects.project", kwargs={"slug": notification.actor.slug}
-                ),
-            }
+            if "new string" in notification.verb and self.profile.custom_homepage:
+                actor = {
+                    "anchor": notification.actor.name,
+                    "url": reverse(
+                        "pontoon.translate.locale.agnostic",
+                        kwargs={
+                            "slug": notification.actor.slug,
+                            "part": "all-resources",
+                        },
+                    )
+                    + "?status=missing,unreviewed,fuzzy",
```

Comment:
I'd prefer that to be `missing` only, since we only send notifications for newly created strings, but I'll let @flodolo make the call. :)

Reply:
Fuzzy don't show up with "missing", and fuzzy are typically added as new strings. So, we need at least missing+fuzzy.

Reply:
Fuzzies are actually updated *existing* strings (which allows us to keep translations).

Reply:
> Fuzzies are actually updated _existing_ strings (which allows us to keep translations).

You mean for Pontoon's internal logic? Because for users they're brand new strings.

Reply:
Correct - that's also why they aren't included in the count shown in the new string notification. Which makes me think including fuzzy there will be confusing.

Reply:
I think they are… I've received a notification from AMO Frontend for 4 new strings this week, and 3 of them were fuzzy.

Reply:
![](https://media.giphy.com/media/wHE6Dd6RCVHQfjK5dy/giphy.gif)

My bad. They are indeed created as new strings, and translations are re-imported and marked as Fuzzy. Sorry for the confusion and thanks for explaining me how my stuff works. :)

Reply:
@eemeli Let's remove `unreviewed` from the list of stati.